src/main/scala/io/github/paoloboni/binance/fapi/parameters/ChangeInitialLeverageParams.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2021 Paolo Boni
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package io.github.paoloboni.binance.fapi.parameters
+
+import io.github.paoloboni.binance.common.OrderSide
+import io.github.paoloboni.binance.fapi._
+import enumeratum.{CirceEnum, Enum, EnumEntry}
+import io.circe.generic.extras.Configuration
+import io.circe.{Decoder, Encoder}
+
+case class ChangeInitialLeverageParams(symbol: String, leverage: Int) {
+  require(leverage > 0 && leverage <= 125)
```

Comment:
can you define leverage as:
```
Type Leverage = Int Refined numeric.Interval.Closed[W.`0`.T, W.`125`.T]
```
and use it here, removing the runtime check
```suggestion
case class ChangeInitialLeverageParams(symbol: String, leverage: Leverage)
```

Reply:
is there a posibilty to also use this `Leverage`type in the response type? because i get a `no EntityDecoder` compiler error when doing this

Reply:
I think so, I can see refined codecs here: https://github.com/circe/circe/blob/master/modules/refined/shared/src/main/scala/io/circe/refined/CirceCodecRefined.scala